openquakeplatform/openquakeplatform/common/templates/common/versions.html

```diff
@@ -0,0 +1,51 @@
+{% extends "site_base.html" %}
+{% load i18n %}
+
+{% block title %} {% trans page_title %} {% endblock title %}
+{% block indexclass %} class="here" {% endblock indexclass %}
+{% block body_class %}bodyclass{% endblock %}
+{% block extra_head %}
+<style>
+  div.versions_content {
+  /* border: 1px solid red; */
+  padding: 16px;
+  }
+
+  div.versions_subcontent {
+  /* border: 1px solid green; */
+  text-align: center;
+  margin-top: 16px;
+  }
+
+  table.versions_table {
+  /* border: 1px solid #cccccc; */
+  margin: 64px auto;
+  }
+
+  table.versions_table td {
+  border-bottom: 1px solid #cccccc;
+  padding: 8px;
+  font-size: 16px;
+  text-align: left;
+  }
+
+  table.versions_table td.second {
+  border-left: 1px solid #cccccc;
+  text-align: right;
+  }
+</style>
+{{ block.super }}
+{% endblock %}
+
+{% block middle %}
+<div class="versions_content">
+  <h2 class="page-title">{{page_title}}</h2>
+  <div class="versions_subcontent">
+    <table class="versions_table">
+      {% for item in apps_list %}
```

Comment:
It's fine as it is, but perhaps you could rename item -> app, just for readability